kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/SessionHandle.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.client.api.v1.dto;
+
+import java.util.Objects;
+import java.util.UUID;
+
+public class SessionHandle {
+  private UUID publicId;
+  private UUID secretId;
+  private int protocolVersion;
+
+  public SessionHandle() {}
+
+  public SessionHandle(UUID publicId, UUID secretId, int protocolVersion) {
+    this.publicId = publicId;
+    this.secretId = secretId;
+    this.protocolVersion = protocolVersion;
+  }
+
+  public UUID getPublicId() {
+    return publicId;
+  }
+
+  public void setPublicId(UUID publicId) {
+    this.publicId = publicId;
+  }
+
+  public UUID getSecretId() {
+    return secretId;
+  }
+
+  public void setSecretId(UUID secretId) {
+    this.secretId = secretId;
+  }
+
+  public int getProtocolVersion() {
+    return protocolVersion;
+  }
+
+  public void setProtocolVersion(int protocolVersion) {
+    this.protocolVersion = protocolVersion;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+    SessionHandle that = (SessionHandle) o;
+    return getProtocolVersion() == that.getProtocolVersion()
+        && Objects.equals(getPublicId(), that.getPublicId())
+        && Objects.equals(getSecretId(), that.getSecretId());
+  }
```

Comment:
we should follow the `org.apache.kyuubi.session.SessionHandle`

Reply:
thanks,

modify it to:

```
  @Override
  public boolean equals(Object o) {
    if (this == o) return true;
    if (o == null || getClass() != o.getClass()) return false;
    SessionHandle that = (SessionHandle) o;
    return Objects.equals(getPublicId(), that.getPublicId())
        && Objects.equals(getSecretId(), that.getSecretId());
  }

  @Override
  public int hashCode() {
    return Objects.hash(getPublicId(), getSecretId());
  }
```